src/Umbraco.Tests.Integration/Testing/UmbracoIntegrationTest.cs

```diff
@@ -254,7 +254,7 @@ protected void UseTestLocalDb(IServiceProvider serviceProvider)
             var databaseFactory = serviceProvider.GetRequiredService<IUmbracoDatabaseFactory>();
 
             // This will create a db, install the schema and ensure the app is configured to run
-            InstallTestLocalDb(databaseFactory, TestHelper.ConsoleLoggerFactory, state, TestHelper.WorkingDirectory);
+            InstallTestLocalDb(databaseFactory, serviceProvider.GetRequiredService<ILoggerFactory>(), state, TestHelper.WorkingDirectory);
```

Comment:
This change should be made in netcore/netcore if pull is rejected for whatever reason.